backend/docs/devices_v1_deployments.oai.yaml

```diff
@@ -0,0 +1,396 @@
+          schema:
+            format: date-time
+            type: string
+        - description: Signature of the URL link
+          in: query
+          name: x-men-signature
+          required: true
+          schema:
+            type: string
+        - description: Device tenant ID
+          in: query
+          name: tenant_id
+          schema:
+            type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                description: Artifact file
+                format: binary
+                type: string
+          description: Successful response
+        "400":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+          description: Invalid Request.
+        "403":
+          content: {}
+          description: The download link has expired or the signature is invalid.
+        "500":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+          description: Internal Server Error.
+      security: []
+      summary: |
+        Internally generated download link for deploying device configurations.
+        All parameters are generated internally when fetching a configuration deployment.
+      tags:
+        - Device API
+components:
+  responses:
+    NotFoundError:
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/Error"
+      description: Not Found.
+    InternalServerError:
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/Error"
+      description: Internal Server Error.
+    InvalidRequestError:
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/Error"
+      description: Invalid Request.
+  schemas:
+    Error:
+      description: Error descriptor.
+      example:
+        error: "failed to decode device group data: JSON payload is empty"
+        request_id: f7881e82-0492-49fb-b459-795654e7188a
+      properties:
+        error:
+          description: Description of the error.
+          type: string
+        request_id:
+          description: Request ID (same as in X-MEN-RequestID header).
+          type: string
+      type: object
+    DeploymentStatus:
+      example:
+        status: success
+      properties:
+        status:
+          enum:
+            - installing
+            - pause_before_installing
+            - downloading
+            - pause_before_rebooting
+            - rebooting
+            - pause_before_committing
+            - success
+            - failure
+            - already-installed
+          type: string
+        substate:
+          description: Additional state information
+          type: string
+      required:
+        - status
+      type: object
+    DeploymentInstructions:
+      example:
```

Comment:
why do we start with an example? we never started in that way, is this part of some practices?